libs/design-system/src/component-audit/ComponentAuditTable.tsx

```diff
@@ -0,0 +1,158 @@
+
+export const ComponentAuditTable: React.FC<ComponentAuditTableProps> = ({ data: jsonData }) => {
+  const [expandedRows, setExpandedRows] = useState<string[]>([]);
+  const [sortColumn, setSortColumn] = useState<SortableKey | null>(null);
+  const [sortOrder, setSortOrder] = useState<'asc' | 'desc' | null>(null);
+
+  const data = useMemo(() => {
+    return Object.entries(jsonData).map(([key, value]) => ({ ...value, name: key.split('/')[1] }));
+  }, [jsonData]);
+
+  const toggleRow = (name: string) => {
+    setExpandedRows((prevState) =>
+      prevState.includes(name) ? prevState.filter((row) => row !== name) : [...prevState, name]
+    );
+  };
+
+  const sortData = (key: SortableKey) => {
+    if (sortColumn === key) {
+      setSortOrder(sortOrder === 'asc' ? 'desc' : 'asc');
+    } else {
+      setSortColumn(key);
+      setSortOrder('asc');
+    }
+  };
+
+  const getSortIcon = (key: SortableKey) => {
+    if (sortColumn !== key) return null;
+    if (sortOrder === 'asc') return '⬆️';
+    if (sortOrder === 'desc') return '⬇️';
+
+    return null;
+  };
+
+  const sortedData = sortColumn
+    ? data.sort((a, b) => {
+        const aValue = a[sortColumn];
+        const bValue = b[sortColumn];
+
+        if (typeof aValue === 'string' && typeof bValue === 'string') {
+          return sortOrder === 'asc' ? aValue.localeCompare(bValue) : bValue.localeCompare(aValue);
+        }
+
+        if (typeof aValue === 'number' && typeof bValue === 'number') {
+          return sortOrder === 'asc' ? aValue - bValue : bValue - aValue;
+        }
+
+        return 0;
+      })
+    : data;
+
+  return (
+    <table className={css({ width: '100%', borderCollapse: 'collapse' })}>
+      <thead>
+        <tr className={css({ bg: 'gray.200', fontSize: '100', fontWeight: 'strong' })}>
+          <th
+            className={css({ py: 2, px: 4, textAlign: 'left', borderBottom: '1px solid #ddd', cursor: 'pointer' })}
+            onClick={() => sortData('name')}
+          >
+            Name {getSortIcon('name')}
+          </th>
+          <th
+            className={css({ py: 2, px: 4, textAlign: 'left', borderBottom: '1px solid #ddd', cursor: 'pointer' })}
+            onClick={() => sortData('instances')}
+          >
+            Instance Count {getSortIcon('instances')}
+          </th>
+          <th
+            className={css({ py: 2, px: 4, textAlign: 'left', borderBottom: '1px solid #ddd', cursor: 'pointer' })}
+            onClick={() => sortData('srcPkg')}
+          >
+            Source {getSortIcon('srcPkg')}
+          </th>
+          <th className={css({ py: 2, px: 4, textAlign: 'left', borderBottom: '1px solid #ddd' })}>Props</th>
+        </tr>
+      </thead>
+      <tbody>
+        {sortedData.map(({ name, instances, props, srcPkg }) => (
+          <React.Fragment key={`${srcPkg}-${name}`}>
+            <tr className={css({ bg: 'surface.page', _even: { bg: '#222222' } })}>
```

Comment:
in the light mode, I see the white/black strips, hard to read
![Screenshot 2024-03-14 at 17 13 24](https://github.com/novuhq/novu/assets/2607232/9f829bf7-df90-4614-a97b-049caa199768)